components/Shop/Item.tsx

```diff
@@ -37,6 +37,10 @@ function Item({ onBuy }: Props) {
       borderRadius={10}
       mt={4}
       mb={4}
+      flexDirection={{
```

Comment:
This made UI bug 
![image](https://user-images.githubusercontent.com/34363701/177193243-0fe733c6-6ab2-4a7a-b66e-ce9347632708.png)